<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="BUDDHIKA-02" failures="1" tests="4" name="Test" time="8.729" errors="0" timestamp="2025-03-15T18:18:44 IST">
  <testcase classname="api.test.UserTests" name="testPostUser" time="2.87"/>
  <testcase classname="api.test.UserTests" name="testGetUserByName" time="1.38"/>
  <testcase classname="api.test.UserTests" name="testUpdateUserByName" time="2.497"/>
  <testcase classname="api.test.UserTests" name="testDeleteUserByName" time="1.294">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTests.testDeleteUserByName(UserTests.java:89)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByName -->
</testsuite> <!-- Test -->
